code/game/gamemodes/heist/heist.dm

```diff
@@ -187,7 +157,6 @@
 
 /datum/game_mode/heist/proc/greet_vox(const/datum/mind/raider)
 	to_chat(raider.current, {"<span class='notice'><B>You are a Vox Raider, fresh from the Shoal!</b>
-The Vox are a race of cunning, sharp-eyed nomadic raiders and traders endemic to Tau Ceti and much of the unexplored galaxy. You and the crew have come to the Exodus for plunder, trade or both.
```

Comment:
Yeah an RP line would be great but literally none of that fits into what players think of vox now or what this PR changes them to.